shopify/mixins.py

```diff
@@ -11,8 +11,15 @@ def count(cls, _options=None, **kwargs):
 
 class Metafields(object):
 
-    def metafields(self):
-        return shopify.resources.Metafield.find(resource=self.__class__.plural, resource_id=self.id)
+    def metafields(self, _options=None, **kwargs):
+        if _options is None:
+            _options = kwargs
+        return shopify.resources.Metafield.find(resource=self.__class__.plural, resource_id=self.id, **_options)
+
+    def metafields_count(self, _options=None, **kwargs):
+        if _options is None:
+            _options = kwargs
```

Comment:
okay cool that makes sense! Sorry I am just not super familiar with python or this repo anymore so figured I would ask.